docs/src/pages/components/data-grid/accessibility/accessibility.md

```diff
@@ -28,20 +28,20 @@ The grid responds to keyboard interactions from the user as well as emitting eve
 
 Use the arrow keys to move the focus.
 
-|                              Keys | Description                                   |
-| --------------------------------: | :-------------------------------------------- |
-|                    <kbd>Tab</kbd> | Navigate between selectable elements          |
-|             <kbd>Arrow Left</kbd> | Navigate between cell elements                |
-|           <kbd>Arrow Bottom</kbd> | Navigate between cell elements                |
-|            <kbd>Arrow Right</kbd> | Navigate between cell elements                |
-|               <kbd>Arrow Up</kbd> | Navigate between cell elements                |
-|                   <kbd>Home</kbd> | Navigate to the first cell of the current row |
-|                    <kbd>End</kbd> | Navigate to the last cell of the current row  |
-| <kbd>CTRL</kbd> + <kbd>Home</kbd> | Navigate to the first cell of the first row   |
-|  <kbd>CTRL</kbd> + <kbd>End</kbd> | Navigate to the last cell of the last row     |
-|                  <kbd>Space</kbd> | Navigate to the next scrollable page          |
-|                <kbd>Page Up</kbd> | Navigate to the next scrollable page          |
-|              <kbd>Page Down</kbd> | Navigate to the previous scrollable page      |
+|                              Keys | Description                                      |
+| --------------------------------: | :----------------------------------------------- |
+|                    <kbd>Tab</kbd> | Navigate between selectable elements             |
```

Comment:
<kbd>Tab</kbd> should move to the next tabbable element, but instead, it moves to the next focusable one. As far as I know, it's wrong.

```suggestion
```

Reply:
I'll handle that one in here #1327